src/TheThingsNetwork.cpp

```diff
@@ -396,19 +402,51 @@ void TheThingsNetwork::autoBaud()
   while (attempts-- && length == 0)
   {
     delay(100);
+    // Send break + Autobaud 
     modemStream->write((byte)0x00);
     modemStream->write(0x55);
     modemStream->write(SEND_MSG);
-    sendCommand(SYS_TABLE, 0, true, false);
-    sendCommand(SYS_TABLE, SYS_GET, true, false);
-    sendCommand(SYS_TABLE, SYS_GET_VER, false, false);
-    modemStream->write(SEND_MSG);
-    length = modemStream->readBytesUntil('\n', buffer, sizeof(buffer));
+    // check we can talk
+    length = checkModuleAvailable();
+    
+    // We succeded talking to the module ?
+    baudDetermined = length > 0 ;
```

Comment:
If a number is non-zero, it will implicitly be true when assigning to a boolean.

Reply:
yes because it's a test operation so it return true or false, I added some parenthesis